### PR TITLE
Fix macos ci

### PR DESCRIPTION
**Pull request by JonasVautherin**
For some reason it started failing with `error: invalid command 'bdist_wheel'`.

### CODE REVIEW

**Comment by JonasVautherin** [comment]
I am a bit puzzled, because it was very painful to get to those two lines. I am not entirely sure if those two lines generally work or if I have just been lucky in the last 4 runs.

At least they make sense, so... let's try with them maybe?

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -125,7 +125,7 @@ jobs:
 
   macOS:
     name: macOS
-    runs-on: macos-latest
+    runs-on: macos-12
```
Comment (julianoes) [inline .github/workflows/main.yml:128]: So according to what I read this should be the same. Although I'm fine pinning this to avoid breakage.

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -140,7 +140,7 @@ jobs:
       - name: Install prerequisites
         run: |
           python3 -m pip install -r requirements.txt
-          python3 -m pip install twine
+          python3 -m pip install twine wheel
```
Comment (julianoes) [inline .github/workflows/main.yml:143]: I've seen that before, probably makes sense.

Why did you remove `twine`? I believe that's required to publish the release, no?

File: .github/workflows/main.yml (hunk 1 above)
Comment (JonasVautherin) [reply .github/workflows/main.yml:128]: Yep, but before I changed it, it was failing :thinking:. So I don't know if something else changed during my tests, or if that is different, but I'm a bit confused.

File: .github/workflows/main.yml (hunk 2 above)
Comment (JonasVautherin) [reply .github/workflows/main.yml:143]: I did not, it's right before `wheel` :laughing:. What time was it when you read that? :grin:

Comment (julianoes) [reply .github/workflows/main.yml:143]: Gosh. Thanks.
